docs/dev/table/python/debugging.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Debugging"
+nav-parent_id: python_tableapi
+nav-pos: 130
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+This page describes how to debug in PyFlink.
+
+* This will be replaced by the TOC
+{:toc}
+
+## Logging Infos
+
+You can take use of Logging module of Python to print infos in Python UDF. e.g.
+
+{% highlight python %}
+
+@udf(input_types=[DataTypes.BIGINT(), DataTypes.BIGINT()], result_type=DataTypes.BIGINT())
+def add(i, j):
+    import logging
+    import sys
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO, format="%(message)s")
+    logging.info("debug")
+    return i + j
+    
+{% endhighlight %}
+
+## Watching Logs
+
+If you set the environment variable `FLINK_HOME`, logs will be placed in the log directory under `FLINK_HOME`.
+Otherwise, logs will be placed in the directory of PyFlink module. You can execute following command to find
+the log directory of PyFlink module:
```

Comment:
```suggestion
If the environment variable `FLINK_HOME` is set, logs will be written in the log directory under `FLINK_HOME`.
Otherwise, logs will be placed in the directory of the PyFlink module. You can execute the following command to find
the log directory of PyFlink module:
```